src/Tickets/Commerce/Admin/Singular_Order_Notices.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * Singular order status notices.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+
+namespace TEC\Tickets\Commerce\Admin;
+
+use Tribe__Admin__Notices;
+
+/**
+ * Class Singular_Order_Notices
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+class Singular_Order_Notices {
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_STATUS_SUCCESSFULLY_UPDATED = 'updated';
+	/**
```

Comment:
These all need a space between them.

```suggestion

	/**
	 * @since TBD
	 *
	 * @var string Notice key.
	 */
	const ORDER_STATUS_SUCCESSFULLY_UPDATED = 'updated';

	/**
```